core/templates/pages/contributor-dashboard-page/translation-opportunities/translation-opportunities.component.ts

```diff
@@ -100,7 +100,9 @@ angular.module('oppia').component('translationOpportunities', {
           },
           controller: 'TranslationModalController'
         }).result.then(function() {
+          ctrl.resetCursorStyle();
```

Comment:
Add a comment before this line, something like:

"Reset the cursor style in case the user had left copy mode on."

Reply:
I've moved the comment here! It should make more sense now, and help improve maintainability.

Reply:
Thanks, can you also move the comment above? Usually comments are written above the code they are commenting on.